### PR TITLE
Update instructions to New Relic Alerting

### PR DESCRIPTION
**Pull request by ronsherf**
New integration instructions to New Relic Legacy Alerting
Jira issue key BUG-229

### CODE REVIEW

File: .idea/jsLibraryMappings.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (erikzaadi) [inline .idea/jsLibraryMappings.xml]: Delete this file from git

File: .idea/libraries/docs_bigpanda_io_node_modules.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<component name="libraryTable">
```
Comment (erikzaadi) [inline .idea/libraries/docs_bigpanda_io_node_modules.xml]: ditto

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (erikzaadi) [inline .idea/vcs.xml]: ditto

**Comment by erikzaadi** [comment]
@rasienko Please have a look at the text on the changes, related to [Bug 229](https://agilepanda.atlassian.net/browse/BUG-229)

**Comment by rasienko** [comment]
@erikzaadi  A few notes…

1. The changes to the last section are a bit confusing. Are we expecting them to create new policies for BigPanda? Or, can they add the BigPanda channel to existing policies? The previous version of the beta instructions suggested that they could add the channel to their existing policies without recreating them.
2. The link to the New Relic docs appears to be a sentence fragment. Try something like, “For more information, see New Relic Alerts documentation.”
3. You can remove the “+” before the New Relic button names. It’s easier to read and still clear. (Also, it looks like Bold didn’t work in some places and there’s an errant “s” after one of these.)
4. Change “Base Url” to “Base URL”.
5. Remove the sentence fragment, “After the Channel successfully created notification appears.”
Let me know if you have questions and when it's ready for another look. Thanks!

**Comment by ronsherf** [comment]
Hi @rasienko the current flow for New Relic integration creation is as follow:
 1. Create notification channel.
2. Create alert policy.
3. Add notification channel to alert policy.

**Comment by rasienko** [comment]
@ronsherf: Thanks for the updates! I don’t seem to have admin access to the OD, so I can’t review the instructions there. Here’s some feedback based on the diff in GitHub.

* In both files, the link to the New Relic docs is missing punctuation. Also, I would link only part of the text and maybe put it after the step. For example, here’s a suggestion for the legacy version. 
`In the New Relic dashboard, navigate to **Notification channels**. For more information, see the [New Relic legacy alerting documentation (https://docs.newrelic.com/docs/alerts/new-relic-alerts/getting-started/alerting-new-relic).`
* In both files, please add punctuation and use a lowercase “c” for, “Under Alerts policy, choose…” (you can also remove “again” from the second instance).
* In the new alerts file, it looks like there’s an extra space so the bold didn’t work in step 2 of the Create Alert Policy section.
* In both files, in the Create Alert Policy section, add “using” after “add all of your conditions by”. 

Let me know if you have questions and when it's ready for another look. Can you grant me access to the OD so I can review it in the product? I’d also like to share it with Stevan, a new product manager on our team who happens to have worked at New Relic! He might have some good advice for us. Thanks again!

**Comment by ronsherf** [comment]
Hi @rasienko, thanks for the comments.
I have fixed your comments and few more staff.
Granted you an admin access, so you suppose to be able to see everything now.
Feel free to share with Stevan, will love to get his feedback.

**Comment by rasienko** [comment]
@ronsherf thanks! I shared the tutorials with Stevan but haven't had a chance to touch base with him yet. Since I'll be at a conference tomorrow, I'll Slack you some screenshots with my editorial recommendations for now. If Stevan has additional feedback, we can get to it next week :)

**Pull request merged**